pkg/graph/schema.resolvers.go

```diff
@@ -1698,6 +1699,89 @@ func (r *mutationResolver) DeleteSystemIntakeContact(ctx context.Context, input
 	}, nil
 }
 
+// SendFeedbackEmail is the resolver for the sendFeedbackEmail field.
+func (r *mutationResolver) SendFeedbackEmail(ctx context.Context, input model.SendFeedbackEmailInput) (*string, error) {
+	var reporterName, reporterEmail string
+
+	if !input.IsAnonymous {
+		euaUserID := appcontext.Principal(ctx).ID()
+		userInfo, err := r.service.FetchUserInfo(ctx, euaUserID)
+		if err != nil {
```

Comment:
Bit of a logic error here; this doesn't actually handle an error, and instead only populates `reporterName` and `reporterEmail` if there _was_ an error. The code should probably be this instead:
```
userInfo, err := r.service.FetchUserInfo(ctx, euaUserID)
if err != nil {
  return nil, err
}
reporterName = userInfo.CommonName
reporterEmail = userInfo.Email.String()
```